pkgs/flox/default.nix

```diff
@@ -40,23 +40,27 @@
       # rather than relying on or modifying the user's `PATH` variable
       NIX_BIN = "${flox-bash}/libexec/flox/nix";
       GIT_BIN = "${gitMinimal}/bin/git";
-      PARSER_UTIL_BIN = parser-util.outPath + "/bin/parser-util";
-      PKGDB_BIN = pkgdb.outPath + "/bin/pkgdb";
-      FLOX_GH_BIN = flox-gh.outPath + "/bin/flox-gh";
-      GH_BIN = gh.outPath + "/bin/gh";
+      PARSER_UTIL_BIN = "${parser-util}/bin/parser-util";
+      PKGDB_BIN = "${pkgdb}/bin/pkgdb";
+      FLOX_GH_BIN = "${flox-gh}/bin/flox-gh";
+      GH_BIN = "${gh}/bin/gh";
 
       # path to bash impl of flox to dispatch unimplemented commands to
       FLOX_SH = "${flox-bash}/libexec/flox/flox";
-      FLOX_SH_PATH = "${flox-bash}";
+      FLOX_SH_PATH = flox-bash.outPath;
```

Comment:
Why change from `foo.outPath`->`${foo}` above and the opposite here?

Reply:
Since it doesn't append the `outPath` I just dropped the string interpolation.
They're equivalent tho.

Reply:
is there any reason to use `.outPath` over string interpolation except visuals?